lib/smart_answer/calculators/minimum_wage_calculator.rb

```diff
@@ -54,7 +54,7 @@ def valid_age_for_living_wage?(age)
     def basic_rate
       rate = @basic_pay / @basic_hours
       if overtime_hours > 0 && overtime_hourly_rate > 0 && rate > overtime_hourly_rate
-        overtime_hourly_rate
+        (rate + overtime_hourly_rate) / 2
```

Comment:
I don't understand what the `2` represents. Please can we avoid introducing more magic numbers.

Reply:
@chao-xian it is the average between the two, this is what I understood from https://govuk.zendesk.com/agent/tickets/2231061 and https://www.gov.uk/overtime-your-rights/overview. I added a comment to make it more clear in the code.

Reply:
Could this be expressed by assigning to a variable named appropriately? Something like `average_rate` and then you wouldn't need to add the comment. How do you feel about that?

Reply:
@chao-xian 👍

Reply:
@chao-xian mmh, Linter complains about that, I simplified the comment